source/09-audit/03-timelines.rst

```diff
@@ -0,0 +1,59 @@
+.. The COUNTER Code of Practice Release 5 © 2017-2023 by COUNTER
+   is licensed under CC BY-SA 4.0. To view a copy of this license,
+   visit https://creativecommons.org/licenses/by-sa/4.0/
+
+Audit Timelines
+---------------
+
+An audit is valid for twelve months from the date of seeding: that is, an audit seeded in April 2025 will be valid until the end of March 2026.
+
+The audit process can be lengthy and report providers should take this into consideration when planning for audits. A typical timeline is shown in table 9.a below.
+
+Table 9.a (below): Audit Timelines
+
+.. only:: latex
+
+   .. tabularcolumns:: |>{\raggedright\arraybackslash}\Y{0.13}|>{\raggedright\arraybackslash}\Y{0.17}|>{\parskip=\tparskip}\Y{0.37}|>{\raggedright\arraybackslash}\Y{0.33}|
+
+.. list-table::
+   :class: longtable
+   :widths: 38 38 24
```

Comment:
The table layout needs to be fixed, like for many other tables that have been changed. This should be done a part of the clean up.